src/components/directory-content.js

```diff
@@ -894,33 +906,35 @@ const DirectoryContent = () => {
                         handleCloseRowMenu();
                 }}
             >
-                {selectedDirectory !== null &&
-                    currentChildren !== null &&
-                    currentChildren.length === 0 && (
-                        <div
-                            style={{ textAlign: 'center', marginTop: '100px' }}
-                        >
-                            <FolderOpenRoundedIcon
-                                style={{ width: '100px', height: '100px' }}
-                            />
-                            <h1>
-                                <FormattedMessage id={'emptyDir'} />
-                            </h1>
-                        </div>
-                    )}
-                <Toolbar>
-                    {allowsDelete(false) && selectedUuids.size > 0 && (
-                        <IconButton
-                            className={classes.icon}
-                            onClick={() => handleOpenDeleteElement()}
-                        >
-                            <DeleteIcon />
-                        </IconButton>
-                    )}
-                </Toolbar>
-                {selectedDirectory !== null &&
-                    currentChildren !== null &&
-                    currentChildren.length > 0 && (
+                {isMetadataLoading ||
```

Comment:
Is removing condition on `selectedDirectory` dangerous ? What is shown when no directory is selected for exemple ?

Reply:
nope, nothing is shown